### PR TITLE
Fix H SAF Soil Moisture Climate Data Record (CDR) reader and documentation

### LINKED ISSUE #8: H109 is not readable
H109 has porosity and ancillary flags in external files and thus can not be read like the H25 products.

@sebhahn I guess they are point data files and should be supported by pynetcf?

### LINKED ISSUE #9: Support reading static layers (porosity and advisory flags)
Add readers for static layers (porosity and advisory flags). It is still open where the data sets will be hosted.

### PR DESCRIPTION
**Pull request by sebhahn**
Simplifying and updating soil moisture climate data record reading routines

Fix #8 
Fix #9

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -114,17 +103,9 @@ Documentation
 Installation
 ============
 
-The packages you have to install depend on the features you want to use. The
-H-SAF image products are disseminated in BUFR (H07, H16, H103, H101, H102, H08)
-or GRIB (H14) format. So to read them you will have to install the appropriate
-packages which will be explained shortly. Unfortunately neither BUFR nor GRIB
-readers work on Windows so if you need these formats then Linux or OS X are your
-only options.
+The packages you have to install depend on the features you want to use. The H SAF soil moisture NRT products are disseminated in BUFR (H16, H103, H101, H102, H08) or GRIB (H14) format. So to read them you will have to install the appropriate packages which will be explained shortly. Unfortunately neither BUFR nor GRIB readers work on Windows so if you need these formats then Linux or OS X are your only options.
```
Comment (cpaulik) [inline README.rst]: Why would you remove the linebreaks?

**Comment by cpaulik** [comment]
Please don't change the complete formatting, variable names and class names in a PR that also includes functionality changes. This is a complete nightmare to review based on the diffs.

File: README.rst (hunk 1 above)
Comment (sebhahn) [inline README.rst]: I usually have a paragraph in one line and the editor takes care of line breaks. Should we break text at 80?

**Comment by sebhahn** [comment]
Sorry, but the package needs to be restructured and this was a first step. In one of my next commits I will move the classes to h_saf.py where they should be.

**Comment by cpaulik** [comment]
This PR should also

Fix #8 
Fix #9

**Comment by sebhahn** [comment]
If the documentation is OK, the branch should be ready

**Comment by sebhahn** [comment]
@cpaulik I think the PR is ready

File: ascat/timeseries.py [hunk 2]
```diff
@@ -1,1113 +1,391 @@
...
+            porosity_hwsd = np.nan
+            data['snow_prob'] = np.nan
+            data['frozen_prob'] = np.nan
+
+        if absolute_sm:
+            # no error assumed for porosity values, i.e. variance = 0
+            por_var = 0.
+
+            data['abs_sm_gldas'] = data['sm'] / 100.0 * porosity_gldas
+            data['abs_sm_noise_gldas'] = np.sqrt(
```
Comment (cpaulik) [inline ascat/timeseries.py]: I think `por_var` should also be squared?

Comment (sebhahn) [inline ascat/timeseries.py]: `por_var` is already the porosity variance

Comment (cpaulik) [inline ascat/timeseries.py]: yes, of course. sorry.

**Comment by cpaulik** [comment]
This was merged manually. Thanks for all your work.

**Pull request closed without merging**